packages/authentication/test/unit/strategy-adapter.test.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {ShimRequest} from '../..';
+import {StrategyAdapter} from '../..';
+import {ParsedRequest} from '@loopback/core';
+
+const passportStrategy = require('passport-strategy');
+
+describe('Strategy Adapter', () => {
+  const mockUser: User = {id: 'mock-user', role: 'mock-role'};
+  interface User {
+    id: string;
+    role: string;
+  }
+
+  describe('authenticate()', () => {
+
+    it('calls the authenticate method of the strategy', (done) => {
```

Comment:
AFAICT, this test runs synchronously, there is no need to use `function (done)` here.